project-base/storefront/graphql/requests/cart/fragments/CartFragment.graphql

```diff
@@ -18,7 +18,8 @@ fragment CartFragment on Cart {
     }
     remainingAmountWithVatForFreeTransport
     transport {
-        ...TransportWithAvailablePaymentsAndStoresFragment
+        ...TransportWithAvailablePaymentsFragment,
+        ...TransportStoresFragment
```

Comment:
With this, you should also change cart.ts:CurrentCartType to this:

`transport: Maybe<TypeTransportWithAvailablePaymentsFragment & TypeTransportStoresFragment>;`

Reply:
This is actually unnecessary as we still have the original fragment `TypeTransportWithAvailablePaymentsAndStoresFragment` (it is used by 2 cypress tests) so I will use this instead.

Reply:
Yeah, I know that we have it and I was even unsure if we still should keep it for the test, but I agree that it is probably okay there. 

However, here I do not think it makes sense and I have to disagree with you. You should not use a type corresponding to a different type than the one you use. If you somehow modify either of the types, it might not be noticed and cause issues in development.